portfolio/banking/banking.go

```diff
@@ -0,0 +1,95 @@
+package banking
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/thrasher-corp/gocryptotrader/currency"
+)
+
+// GetBankAccountByID Returns a bank account based on its ID
+func GetBankAccountByID(id string) (*Account, error) {
+	m.Lock()
+	defer m.Unlock()
+
+	for x := range Accounts {
+		if strings.EqualFold(Accounts[x].ID, id) {
+			return &Accounts[x], nil
+		}
+	}
+	return nil, fmt.Errorf(ErrBankAccountNotFound, id)
+}
+
+// ExchangeSupported Checks if exchange is supported by bank account
+func (b *Account) ExchangeSupported(exchange string) bool {
+	exchList := strings.Split(b.SupportedExchanges, ",")
+	for x := range exchList {
+		if exchList[x] == exchange {
```

Comment:
Please use strings.EqualFold here

Reply:
oh yeah we also have StringDataCompareInsensitive always forget about that